src/workloads/execution/GroupSpillToDisk.yml

```diff
@@ -0,0 +1,126 @@
+- Name: InsertData
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          firstField: {^RandomInt: {min: 1, max: *FirstFieldUniqueValCount}}
+          secondField: {^RandomString: {length: *RandomStrLength, alphabet: "0123456789"}}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+
+# This query has a small number of groups, but each group is a very large set.
+- Name: AddToSetAccumulatorWithLargeSets
+  Type: CrudActor
+  Database: *Database
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$group: {_id: "$firstField", list: {$addToSet: "$secondField"}}}]
+            Options:
+              AllowDiskUse: true
+
+# Similar to the previous query, but using $push rather than $addToSet.
+- Name: PushAccumulatorConstructingLargeArrays
+  Type: CrudActor
+  Database: *Database
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$group: {_id: "$firstField", list: {$push: "$secondField"}}}]
+            Options:
+              AllowDiskUse: true
+
+# This query groups by "secondField", which has many distinct values. Spilling is required since
+# there are many groups, but the state for each individual group is small.
+- Name: SumAccumulatorSpillDueToManyGroups
+  Type: CrudActor
+  Database: *Database
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [4]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$group: {_id: "$secondField", count: {$sum: 1}}}]
+            Options:
+              AllowDiskUse: true
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      # TODO PERF-3480: Enable this workload against 'standalone', 'standalone-all-feature-flags'
+      # and 'standalone-sbe' once the SBE performance improves. These queries will use SBE unless
+      # classic is forced, but they run for too long at the moment due to the performance problem
+      # reported in SERVER-70395.
+      - standalone-classic-query-engine
```

Comment:
Could you reconfigure your patch build to run 'auto run' on the standalone classic variant: https://spruce.mongodb.com/version/635ad15c5623435c5fb6d9a3/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC ?

Reply:
D'oh! My bad, thanks for catching this. Done, hopefully it passes!